File: Linkpoint/src/main/java/com/linkpoint/slproto/modules/rlv/commands/RLVCmdShowInventory.java
package com.linkpoint.slproto.modules.rlv.commands;

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType;

public class RLVCmdShowInventory extends RLVCmdGenericRestriction {
    public RLVCmdShowInventory() {
        super(RLVRestrictionType.showinv, false);
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/slproto/modules/rlv/commands/RLVCmdShowInventory.kt
package com.linkpoint.slproto.modules.rlv.commands

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType

class RLVCmdShowInventory : RLVCmdGenericRestriction() {
    public RLVCmdShowInventory() {
        super(RLVRestrictionType.showinv, false)
    }
}
